openpype/hosts/maya/plugins/publish/convert_gltf_shader.py

```diff
@@ -0,0 +1,78 @@
+import os
+from maya import cmds
+import pyblish.api
+
+from openpype.pipeline import publish
+
+
+class ConvertGLSLShader(publish.Extractor):
+    """
+    Converting StingrayPBS material to GLSL Shaders
+    specially for the glb export through Maya2GLTF plugin
+
+    """
+    order = pyblish.api.ExtractorOrder - 0.1
+    hosts = ["maya"]
+    label = "Convert StingrayPBS to GLTF"
+    families = ["gltf"]
+    optional = True
+
+    def process(self, instance):
+        meshes = cmds.ls(instance, type="mesh", long=True)
+        self.log.info("meshes: {}".format(meshes))
+        # load the glsl shader plugin
+        cmds.loadPlugin("glslShader.mll", quiet=True)
```

Comment:
I believe this will be Windows-only due to the `.mll` suffix. You should be able to load these without the suffix. For other OS'es it would then load e.g. the `.so` file.

```suggestion
        cmds.loadPlugin("glslShader", quiet=True)
```